src/helper/object.js

```diff
@@ -103,16 +103,21 @@ function toSnakeCase(object, exceptions) {
   }, {});
 }
 
-function toCamelCase(object, exceptions) {
+function toCamelCase(object, exceptions, options) {
```

Comment:
Do you really need all the options passed in? Or just boolean `keepCasing`?

Reply:
I'm not passing all the options, it's a new options object and is meant to be internal only (the regular WebAuth options are not passed to this method)

Reply:
Understood but does it need a whole options parameter or would a flag work fine here? How much more would you want this function to do?

Reply:
it would work today, but an options object makes it future proof

Reply:
It could make it future-overloaded as well ...

Reply:
I also think using options is better than flags because of this:

`toCamelCase(authResult, ['id_token'], true)`

What does the array mean? What does the true mean? If you don't read the implementation, you'll never know.